Makefile

```diff
@@ -3,13 +3,15 @@ bandit:
 	bandit -r ./create_aio_app -x create_aio_app/template -s B101
 
 checkrst:
-	python setup.py check --restructuredtext
+	python -c 'import setuptools; setuptools.setup()' check
```

Comment:
Running `setup.py check` is deprecated. Use `twine check` instead.

Reply:
Replaced with twine.